### ISSUE TITLE
Shall we have a Wechat group?

### ISSUE THREAD

**Issue by RobinSeaside** [lzjqsdd/scikit-learn-doc-cn#31]
Sklearn is very important to machine learners. To better push the translation forward, I propose that we create a Wechat group.

My account: Awang-tj

Have a nice day.

**Comment by minoriwww** [comment]
How about telegram? btw I'v added u on wechat =) 
@lzjqsdd 

Regards.

**Comment by RobinSeaside** [comment]
Yeah. Telegram is a wise choice. Cheers!

**Comment by RobinSeaside** [comment]
training set: 训练集
test set: 测试集
examples: 样例

**Comment by lzjqsdd** [comment]
my account :1315172728

**Comment by RobinSeaside** [comment]
@minoriwww I lost your contact number. Would you please add me again?

**Comment by lzjqsdd** [comment]
@RobinSeaside  https://github.com/lzjqsdd/scikit-learn-doc-cn/invitations

**Comment by KratosWang** [comment]
recently i am learning sklearn ,is so great to have a translated documents. i'd love to join in your translate team but the Wechat 2-D code is overdue，could you upload a new one ? THX

**Comment by lzjqsdd** [comment]
already updated

**Comment by ibaihuo** [comment]
@lzjqsdd  you should update the 2-D code every 7 days。

I want to contribute the doc.

**Comment by goodbai-nlp** [comment]
The Wechat 2-D code is overdue, could you upload a new one? :)

**Comment by wxianfeng** [comment]
@minoriwww pls add me to wechat group, 3kx

wechat account: wxianfeng
